css/index.css

```diff
@@ -3,144 +3,395 @@
+  width: 100%;
+  margin: 2% 0;
+  font-size: 1.2rem;
 }
 
+.top-content img {
+  width: 106%;
+  margin: 0 -3%;
+}
+
+/* Middle Content */
+
 .middle-content {
-    margin-bottom: 20px;
-    border-bottom: 1px dashed black;
+  margin: 0 3%;
+  border-bottom: 1px dashed black;
 }
 
 .middle-content h2 {
-    padding: 0 2%;
-    margin-bottom: 0;
+  padding: 0 2%;
+  margin-bottom: 0;
 }
 
 .middle-content .boxes {
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-evenly;
+  display: grid;
+  grid-template-columns: 25% 25% 25%;
+  justify-content: space-around;
 }
 
 .middle-content .boxes .box {
-    width: 12.5%;
-    height: 100px;
-    background: black;
-    margin: 20px 2.5%;
-    color: white;
-    display: flex;
-    align-items: center;
-    justify-content: center;
+  width: 100%;
+  height: 100px;
+  background: black;
+  margin: 20px 0;
+  color: white;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
 
+/* Bottom Content */
+
 .bottom-content {
-    display: flex;
-    margin: 0 2% 20px;
-    justify-content: space-around;
+  display: grid;
+  margin: 5% 2%;
+  grid-template-columns: 1fr;
+  justify-content: space-around;
 }
 
 .bottom-content .text-container {
-    padding-right: 4%;
+  padding-right: 4%;
 }
 
 .bottom-content .text-container:last-child {
-    padding-right: 0;
+  padding-right: 0;
 }
 
 footer {
-    width: 100%;
-    background: black;
+  width: 100%;
+  background: black;
 }
 
 footer nav {
-    width: 60%;
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    padding: 20px 2%;
-    font-size: 14px;
+  width: 60%;
+  display: flex;
+  margin: 0 auto;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 2%;
+  font-size: 1.4rem;
 }
 
 footer nav a {
-    color: white;
-    text-decoration: none;
+  color: white;
+  text-decoration: none;
+}
+
+@media(min-width:401px) and (max-width: 768px) {
+  .container {
+    max-width: 768px;
+    margin: 0%;
+  }
+  section.top-content,
+  section.middle-content,
+  section.bottom-content {
+    margin-left: 0;
+    margin-right: 0;
+  }
+  /* Header */
+  header {
+    max-width: 768px;
+    grid-template-rows: 1fr 1fr;
+    height: auto;
+  }
+  header img {
+    margin: 1% auto;
+  }
+  nav ul {
+    grid-template-rows: 1fr;
+    grid-template-columns: repeat(5, 1fr);
+    justify-content: space-around;
+  }
+  nav ul a {
+    border: none;
+  }
+  /* Top Content */
+  .top-content {
+    grid-template-columns: 1fr 1fr;
+    padding: 1% 2%;
+  }
+  .top-content .text-container {
+    padding: 3% 0;
+  }
+  .top-content img {
+    grid-column: 1 / 3;
+  }
+  /* Middle Content */
+  .middle-content .boxes {
+    grid-template-columns: repeat(5, 13%)
+  }
+  .middle-content h2 {
+    margin-top: 2%;
+  }
+  /* Bottom Content */
+  .bottom-content {
+    grid-template-columns: repeat(3, 25%);
+  }
+  /* footer */
+  footer nav {
+    margin: 0 auto 0 0;
```

Comment:
In comparison to the design image, the footer should be aligned to the left as well! And needs some padding around each `a` component 😄